src/eda_server/db/models/job.py

```diff
@@ -35,6 +36,26 @@
     sa.Column("uuid", postgresql.UUID),
 )
 
+
+activation_job_instances = sa.Table(
```

Comment:
Since it's a new table, where it's populated?

Reply:
For Activation Instance, its job instance table is populated on activation instance creation but for Activation, we currently don't have that function to populate its job instance table.